### DISCUSSION TITLE
Adding Gifsicle to the Tea package manager

### DISCUSSION THREAD

**Discussion by jacob-willden** [kohler/gifsicle#192] [General]
Gifsicle is available to install via [Homebrew](https://brew.sh/) (and I'm personally very grateful for that, since Gifsicle has been highly helpful to me). I think it would be a good idea to add Gifsicle to [Tea](https://tea.xyz/), a new package manager from the people behind Homebrew meant to make package management completely cross-platform and self-contained. Since [writing the YAML file needed to port the software to Tea](https://github.com/teaxyz/pantry/wiki) requires at least a fair amount of knowledge about how Gifsicle is built, I wanted to start a discussion about it. Any thoughts?

**Comment by kohler** [comment]
I am entirely happy for someone else to write this :)

**Comment by jacob-willden** [comment]
My bad, looks like someone already added [Gifsicle in Tea's package list](https://github.com/teaxyz/pantry/blob/main/projects/lcdf.org/gifsicle/package.yml) 2 weeks ago!
